tests/sampling/test_jax.py

```diff
@@ -49,6 +52,14 @@ def test_old_import_route():
     assert set(new_sj.__all__) <= set(dir(old_sj))
 
 
+def test_jax_PosDefMatrix():
+    x = pt.tensor()
+    matrix_pos_def = PosDefMatrix()
+    x_is_pos_def = matrix_pos_def(x)
+    out_fg = FunctionGraph(outputs=[x_is_pos_def])
+    compare_jax_and_py(fgraph=out_fg, test_inputs=pt.eye(3))
```

Comment:
@ricardoV94 I can not use `from tests.link.jax.test_basic import compare_jax_and_py` from `pymc` (because it belongs to the tests) ...shall we add a copy?

Reply:
I don't think there's anything special about that. I would just compile a function to JAX and confirm it returns the expected results

Reply:
I want to get the test working with a simple case, and then I will add parametrize it with edge cases.

Reply:
We should compile only once and test with multiple values (pos and not pos). Parametrize is not ideal because we don't want to recompiles/jit just to test a new value